lib/ts/recipe/emailverification/components/features/emailVerification/index.tsx

```diff
@@ -29,8 +29,9 @@ import { ComponentOverrideContext } from "../../../../../components/componentOve
 import { SessionContextType, SessionContext } from "../../../../session";
 import { defaultTranslationsEmailVerification } from "../../themes/translations";
 import { RecipeInterface } from "supertokens-web-js/recipe/emailverification";
+import UserContextWrapper from "../../../../../usercontext/userContextWrapper";
 
-type Prop = FeatureBaseProps & { recipe: Recipe; userContext: any };
+type Prop = FeatureBaseProps & { recipe: Recipe; userContext?: any };
```

Comment:
Props of other feature components don't have `userContext?: any`. So why does this one? Please be consistent

Reply:
EmailVerification is the only feature component that is a class component. I chose to not convert it into a functional one because the changes for integrating web-js are already widespread